Paint/Paint/MainForm.cs

```diff
@@ -0,0 +1,153 @@
+            lastImage = new Bitmap(newBitmapScale.X, newBitmapScale.Y);
+            UpdateImage();
+        }
+
+        private void ClearImage(object sender, EventArgs eventArg)
+        {
+            UpdateImage();
+        }
+
+        private void OpenOpeningImageDialog(object sender, EventArgs eventArg)
+        {
+            if (OpenNewImageDialog.ShowDialog() == DialogResult.Cancel) return;
+            lastImage = new Bitmap(OpenNewImageDialog.FileName);
+            UpdateImage();
+        }
+
+        private void OpenSavingImageDialog(object sender, EventArgs eventArg)
+        {
+            if (SaveNewImageDialog.ShowDialog() == DialogResult.Cancel) return;
+            CanvasPB.Image?.Save(SaveNewImageDialog.FileName);   
+        }
+        
+        private void CoordinatesSetter(object sender, MouseEventArgs eventArg)
+        {
+            CoordinatesLabel.Text = $@"({eventArg.X}, {eventArg.Y})";
+        }
+        
+        private void CoordinatesClearer(object sender, EventArgs eventArg)
+        {
+            CoordinatesLabel.Text = "";
+        }
+
+        private void SetNewColor(object sender, EventArgs eventArg)
+        {
+            ColorPickerDialog.ShowDialog();
+            currentPenColor = ColorPickerDialog.Color;
+            ColorPickerBtn.BackColor = currentPenColor;
+            drawingPen.Color = currentPenColor;
+        }
+
+        private void DrawByPressedPen(object sender, MouseEventArgs eventArg)
+        {
+            if (isPenDown && lastPoint != Point.Empty)
+            {
+                using (Graphics g = Graphics.FromImage(CanvasPB.Image))
+                {
+                    g.DrawLine(drawingPen, lastPoint, eventArg.Location);
+                }
+                CanvasPB.Invalidate();
+                lastPoint = eventArg.Location;
+            }
+        }
+
+        private void OnMouseDown(object sender, MouseEventArgs eventArg)
+        {
+            isPenDown = true;
+            lastPoint = eventArg.Location;
+        }
+        
+        private void OnMouseUp(object sender, MouseEventArgs eventArg)
+        {
+            isPenDown = false;
+            lastPoint = Point.Empty;
+        }
+
+        private void UpdateImage()
+        {
+            CanvasPB.Image = new Bitmap(lastImage);
+            CanvasPB.Size = new Size(lastImage.Width, lastImage.Height);
+        }
+        
+        private void UpdatePen(object sender, EventArgs e)
+        {
+            drawingPen.Color = currentPenColor;
+            penWidth = (int)PenSizeUD.Value;
+            drawingPen.Width = penWidth;
+        }
+        
+        private Point ShowCustomDialog()
+        {
+            Form prompt = new Form();
+            prompt.Width = 200;
+            prompt.Height = 150;
+            prompt.Text = "New Image";            
+            prompt.AutoSize = false;
+            prompt.MaximizeBox = false;
+            prompt.FormBorderStyle = FormBorderStyle.FixedSingle;
```

Comment:
Для диалогов так лучше
```suggestion
            prompt.FormBorderStyle = FormBorderStyle.FixedDialog;
```